themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -116,6 +116,9 @@ JS;
       <?php endif; ?>
       <a class="sr-only" href="#content"><?=$this->transEsc('Skip to content') ?></a>
       <?=$this->render('header.phtml')?>
+      <div id="browser-outdated" class="container-fluid system-message hidden">
+        <?=$this->translate('outdated_browser_message_html');?>
+      </div>
```

Comment:
It's concerning to me that this message will be on every page without an option to dismiss. Also, moving or removing this message will require changing `layout.phtml`. Can we add a configuration option?